recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.0.2" %}
+{% set sha256 = "37d563e36a44a5309b2f53b394d7391d49c8a62b0e8d0f0d13f54ea4e2943230" %}
+# {% set md5 = "b1432aa11b6d244c4fac61737c0c9d95" %}
+
+package:
+  name: obspy
+  version: {{ version }}
+
+source:
+  fn: obspy-{{ version }}.zip
+  url: https://pypi.io/packages/source/o/obspy/obspy-1.0.2.zip
+  sha256: {{ sha256 }}
+#  url: https://www.geophysik.uni-muenchen.de/%7Emegies/obspy-{{ version }}.zip
+#  md5: {{ md5 }}
+
+
+build:
+  number: 0
+  preserve_egg_dir: yes
+  detect_binary_files_with_prefix: true
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Can I do the same for `conda-build`, i.e. add it here in the `meta.yaml` with a fixed version number? That way I would not need any patches to the CI scripts and we would be all good to go..